src/test/java/io/managed/services/test/rhoas/KafkaRhoasBasicTests.java

```diff
@@ -517,6 +526,35 @@ public void testDeleteServiceAccount() {
             () -> cli.describeServiceAccount(serviceAccount.getClientId()));
     }
 
+    @Test(dependsOnMethods = "testLogin", enabled = true)
+    @SneakyThrows
+    public void testCreateKafka() {
+        LOGGER.info("Create Kafka instance with name {}", KAFKA_CREATE_INSTANCE_NAME);
+        var k = cli.createKafka(KAFKA_CREATE_INSTANCE_NAME);
+
+        LOGGER.info("Wait for Kafka instance with name: {}, with id: {}", k.getName(), k.getId());
+        kafka = CLIUtils.waitUntilKafkaIsReady(cli, k.getId());
```

Comment:
also (not sure but asked DevExp) it seems that kafka instnace does not have to be in ready state in order to change its owner. Same goes for change in reauthentication. 
So this could possibly speed up things a bit.

Reply:
Fixed